docs-ref-services/latest/service-bus.md

```diff
@@ -7,9 +7,7 @@ manager: timlt
 ms.date: 03/04/2020
 ms.topic: article
 ms.prod: azure
-ms.technology: azure
 ms.devlang: nodejs
-ms.service: Service Bus
```

Comment:
Can you share why we had to remove the `ms.service` attribute? Just trying to understand so that we don't re-introduce these changes :)